Mathlib/Data/IsROrC/Basic.lean

```diff
@@ -53,6 +53,25 @@ open ComplexConjugate
 
 /--
 This typeclass captures properties shared by ℝ and ℂ, with an API that closely matches that of ℂ.
+
+While the name of the class begins with `Is`,
+this typeclass carries data (all operations common to real and complex numbers),
+not just claims existence of these operations.
+
+This class has exactly two instances: one for `ℝ` and one for `ℂ`.
+In particular, there is no instance for `ULift K`.
+This is done to make it possible to use` [IsROrC K] [NormedSpace K E]`
+as an assumption of an instance.
```

Comment:
I don't know what "as an assumption of an instance" means :-(

Reply:
See #10524 for an example (I hope, it'll work). Suggestions about better wording are very welcome.